sim/warrior/items.go

```diff
@@ -315,13 +318,18 @@ func (warrior *Warrior) ApplyRegicideWarriorEffect(itemID int32, aura *core.Aura
 		Handler: func(sim *core.Simulation, spell *core.Spell, result *core.SpellResult) {
 			debuff := debuffAuras.Get(result.Target)
 			debuff.Activate(sim)
+			// Activate aura for APL usage when effect is triggered
+			// Implementation of cast condition is below in 'ApplyOnInit'
+			if debuff.GetStacks() == 20 {
+				regicideAura.Activate(sim)
+			}
```

Comment:
I imagine that we'd want to do this after the `AddStack` below? this would only activate the aura after proccing again after reaching 20 on the previous proc